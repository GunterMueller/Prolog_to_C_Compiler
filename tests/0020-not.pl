main :-
	(\+true; display('1 ok\n')),
	\+fail, display('2 ok\n'),
	foo.

foo :- \+(!).
foo :- display('foo\n').
