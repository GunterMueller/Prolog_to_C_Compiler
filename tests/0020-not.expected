1 ok
2 ok
foo
